packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -54,39 +55,42 @@ const formatComparisonValue = (
     }
     switch (format) {
         case ComparisonFormatTypes.PERCENTAGE:
-            return `${prefix}${formatValue(value, {
-                format: Format.PERCENT,
+            return `${prefix}${applyCustomFormat(value, {
                 round: 0,
+                type: CustomFormatType.PERCENT,
             })}`;
         case ComparisonFormatTypes.RAW:
             if (item !== undefined && isTableCalculation(item)) {
-                return `${prefix}${formatTableCalculationValue(
-                    item.format,
-                    value,
-                )}`;
+                return `${prefix}${formatItemValue(item, value)}`;
```

Comment:
same here ^

Reply:
Same reason, but here we're passing `item.format` (legacy) which isn't necessarily percent.